src/main/resources/static/js/subscription.js

```diff
@@ -495,6 +492,31 @@ jQuery(document).ready(function () {
         table.responsive.recalc();
     });
 
+    function isEditAndDeleteButtonsDisabled(row, data) {
+        var disableEditDeleteButtons = true
+        var currentUser = sessionStorage.getItem("currentUser");
+
+        var isAnonymousSubscription = row.ldapUserName == null || row.ldapUserName == undefined || row.ldapUserName.length == 0
+        var isCurrentUserLoggedIn = currentUser != null && currentUser != undefined && currentUser.length != 0
+
+        if (isSecured == false || isAnonymousSubscription == true) {
```

Comment:
What is isSecured? is it if Ldap is enabled?

Reply:
isSecured is renamed to ldapEnabled.